include/core/JamomaSample.h

```diff
@@ -44,8 +44,9 @@ namespace Jamoma {
 	class SampleBundle {
 		using SampleBundleData = std::vector<SampleVector>;
 		
-		SampleBundleData	mChannels;		// each item in this vector represents a channel, which itself is a vector of samples
-		size_t				mFrameCount;	// if we change the number of channels we could loose the framecount in the vectors contained by mChannels, so we cache it here.
+		SampleBundleData	mChannels;		//!< each item in this vector represents a channel, which itself is a vector of samples
+		size_t				mFrameCount;	//!< if we change the number of channels we could loose the framecount in the vectors contained by mChannels, so we cache it here.
+        size_t              mPaddingAmount = 0;     //!< if padding has been added, this value is used to track the number of extra Frames at the beginning and end
```

Comment:
@nwolek on further reflection, I'm thinking we should have larger default padding size.  At least 4.  The desire to play back with cubic interpolation would suggest that as the absolute minimum.  For reference, in Max the buffer~ uses 16 samples of padding by default.